packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -406,6 +409,14 @@ function ViewGroupInner(
                     quantity: group.contactIds.length,
                   })}
             </div>
+            <div className='group-member-filter'>
+              <SearchInput
+                id='group-member-filter'
+                onChange={e => setMemberFilter(e.target.value)}
+                value={memberFilter}
+                onClear={() => setMemberFilter('')}
```

Comment:
It's quite laggy for big groups. Have you considered [`useDeferredValue`](https://react.dev/reference/react/useDeferredValue) or a debounce?

Reply:
Solved